ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/editorhook/js/panelselect.js

```diff
@@ -0,0 +1,42 @@
+/*******************************************************************************
+ * Copyright 2023 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+(function(ns) {
+
+    var PanelSelector = CQ.CoreComponents.PanelSelector;
+
+    if (PanelSelector) {
+        var getPanelTitle = PanelSelector.prototype.getTitle;
+        PanelSelector.prototype.getTitle = function (editable, item, index) {
+            var title = "<span class='foundation-layout-util-subtletext cmp-panelselector__indexMarker'>" + index + "</span>&nbsp;&nbsp;";
+            var subTitle = "";
+
+            title = title + " " + Granite.I18n.getVar(ns.editableHelper.getEditableDisplayableName(editable));
+
+            if (item && item.label && item.label.value) {
```

Comment:
This might break for SITES components if form container is used inside SITES editor, you still have to also fallback to cq:panelTitle as done in the original implementation

Reply:
There is already fallback condition on line 36